C/Makefile

```diff
@@ -32,20 +32,21 @@ all:build
 
 build: $(OBJS)
 	@$(CC) $(CCFLAGS) $(OBJS) -o $(ELF_TARGET)
-	avr-objcopy -j .text -j .data -O ihex $(ELF_TARGET) $(HEX_TARGET)
+	@avr-objcopy -j .text -j .data -O ihex $(ELF_TARGET) $(HEX_TARGET)
 	@echo "\n[Makefile] : Compiled successfully."
 
-%.o : %.cpp
+%.o : %.c
 	@$(CC) $(CCFLAGS) $(INC) -c $< -o $@
 	@$(eval SOURCES_CTR=$(shell echo $$(($(SOURCES_CTR)+1))))
-	@printf '\033[2K\r%s' "[Makefile] : [$(SOURCES_CTR)/$(words $(SOURCES))] $<"
+	printf '\033[2K\r%s' "[Makefile] : [$(SOURCES_CTR)/$(words $(SOURCES))] $<"
```

Comment:
Did you remove `@` symbol to try something or you mean to remove it forever?

Reply:
To try something and I forgot to add it back